sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/AssistantsClient.java

```diff
@@ -3765,4 +3766,61 @@ public PageableList<VectorStoreFile> listVectorStoreFileBatchFiles(String vector
         return PageableListAccessHelper.create(vectorStoreFileList.getData(), vectorStoreFileList.getFirstId(),
             vectorStoreFileList.getLastId(), vectorStoreFileList.isHasMore());
     }
+
+    /**
+     * Create a vector store file by attaching a file to a vector store.
+     *
+     * @param vectorStoreId The ID of the vector store for which to create a File.
+     * @param fileId A File ID that the vector store should use. Useful for tools like `file_search` that can access
+     * files.
+     * @param chunkingStrategy The chunking strategy used to chunk the file(s). If not set, will use the auto strategy.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return description of a file attached to a vector store.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public VectorStoreFile createVectorStoreFile(String vectorStoreId, String fileId,
```

Comment:
Are these new generated convenience methods?

Reply:
Yeah. All of Changes in source code are generated except for Tests